test/use-package-test.el

```diff
@@ -1,4 +1,4 @@
-;;; use-package-tests.el --- Tests for use-package.el
+;;; use-package-test.el --- Tests for use-package.el
```

Comment:
Thanks for the example, I thought it would be a mix of both suffixes. Since this might eventually be integrated into Emacs, I'll see if ert-runner takes the `-tests.el` suffix.